README.md

```diff
@@ -117,9 +145,121 @@ Get all reports for the user:
 Get-PowerBIReport
 ```
 
+If you are a tenant administrator, you can view all reports in your tenant by using assigning `-Scope Organization`:
+
+```powershell
+Get-PowerBIReport -Scope Organization
+```
+
+### Get Dashboards
+
+Get dashboards for the user:
+
+```powershell
+Get-PowerBIDashboard
+```
+
+If you are a tenant administrator, you can view all dashboards in your tenant by adding `-Scope Organization`:
+
+```powershell
+Get-PowerBIDashboard -Scope Organization
+```
+
+### Get Tiles
+
+Get tiles within a dashboard:
+
+```powershell
+Get-PowerBITile -DashboardId 9a58d5e5-61bc-447c-86c4-e221128b1c99
+```
+
+### Get Imports
+
+Get Power BI imports:
+
+```powershell
+Get-PowerBIImport
+```
+
+### Create Report
+
+Create a report in Power BI by uploading a *.pbix file:
+
+```powershell
+New-PowerBIReport -Path .\newReport.pbix -Name 'New Report'
+```
+
+By default, the report is placed in the user's My Workspace. To place in a different workspace, use the `-WorkspaceId` or `-Workspace` parameters:
+
+```powershell
+New-PowerBIReport -Path .\newReport.pbix -Name 'New Report' -WorkspaceId f95755a1-950c-46bd-a912-5aab4012a06d
+```
+
+### Export Report
+
+Export a Power BI report to *.pbix file:
+
+```powershell
+Export-PowerBIReport -Id b48c088c-6f4e-4b7a-b015-d844ab534b2a -OutFile .\exportedReport.pbix
+```
+
+If the workspace exists outside the My Workspace, export with the `WorkspaceId` or `-Workspace` parameter:
+
+```powershell
+Export-PowerBIReport -Id b48c088c-6f4e-4b7a-b015-d844ab534b2a -OutFile .\exportedReport.pbix -WorkspaceId 3bdd9735-0ab5-4f21-bd5d-87e7f1d7fb84
+```
+
+### Get Datasets
+
+Get Power BI datasets:
+
+```powershell
+Get-PowerBIDataset
+```
+
+### Get Datasources
+
+Get Power BI datasources for a dataset:
+
+```powershell
+Get-PowerBIDatasource -DatasetId 65d7d7e5-8af0-4e94-b20b-50a882ae15e1
+```
+
+### Get Tables
+
+Get Power BI tables contained within a dataset:
+
+```powershell
+Get-PowerBITable -DatasetId 65d7d7e5-8af0-4e94-b20b-50a882ae15e1
+```
+
+### Call the Power BI Rest API
+
+As the cmdlets haven't been built to handle all the scenarios the [Power BI API](https://docs.microsoft.com/en-us/rest/api/power-bi/) supports, you can reuse the authenticated session from `Connect-PowerBIServiceAccount` to make custom REST requests:
```

Comment:
"For [Power BI APIs](https://docs.microsoft.com/en-us/rest/api/power-bi/) that lack corresponding cmdlets, you can reuse the authenticated session from `Connect-PowerBIServiceAccount` to make custom REST requests"